sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/protobuf/MetricsImporter.java

```diff
@@ -81,13 +82,23 @@ void consumeFor(InputFile inputFile, MetricsInfo message) {
       saveMetric(context, inputFile, CoreMetrics.COGNITIVE_COMPLEXITY, message.getCognitiveComplexity());
     }
 
-    fileLinesContext.save();
+    if (isSonarQubeGreaterThanOrEqualTo62()) {
+      for (Integer executableLineNumber : message.getExecutableLinesList()) {
+        fileLinesContext.setIntValue(CoreMetrics.EXECUTABLE_LINES_DATA_KEY, executableLineNumber, 1);
+      }
+
+      fileLinesContext.save();
```

Comment:
Is this _fileLinesContext.save();_ inside the correct set of brackets - shouldn't it be after line 91?

Also, if that's the case shouldn't some of the tests have failed?

Reply:
2x Yes. I haven't re-run all combinatinons relying on the build server. The current qa failure is because of this problem.